src/breakpoints.jl

```diff
@@ -189,19 +189,18 @@ end
 
 function prepare_slotfunction(framecode::FrameCode, body::Union{Symbol,Expr})
     uslotnames = Set{Symbol}()
-    slotnames  = Symbol[]
-    for name in framecode.src.slotnames
-        if name ∉ uslotnames
-            push!(slotnames, name)
-            push!(uslotnames, name)
-        end
-    end
     framename, dataname = gensym("frame"), gensym("data")
     assignments = Expr[:($dataname = $framename.framedata)]
     default = Unassigned()
-    for i = 1:length(slotnames)
-        slotname = framecode.src.slotnames[i]
-        qslotname = QuoteNode(slotname)
+    for slotname in framecode.src.slotnames
+        if slotname === Symbol("")
```

Comment:
Not entirely sure whether this is correct, but our tests at least say that it's fine.

Reply:
Would things pass even if this is removed?

Reply:
Yep. Removing the check ;)